clinica/pipelines/t1_volume_tissue_segmentation/t1_volume_tissue_segmentation_pipeline.py

```diff
@@ -88,7 +88,7 @@ def build_input_node(self):
                                             self.bids_directory,
                                             T1W_NII)
         except ClinicaException as e:
-            err = 'Clinica faced error(s) while trying to read files in your CAPS directory.\n' + str(e)
+            err = 'Clinica faced error(s) while trying to read files in your BIDS directory.\n' + str(e)
```

Comment:
Just to confirm... this change isn't related to the goal of this PR, isn't ? (It seems like a forgotten typo from a previous PR)

Reply:
When updating the Jinja2 using `t1-volume-tissue-segmentation` pipeline as a model, I noticed this typo. I forgot to make an atomtic commit.